scene.go

```diff
@@ -87,7 +87,6 @@ func SetScene(s Scene, forceNewWorld bool) {
 
 		wrapper.mailbox.listeners = make(map[string][]MessageHandler)
 
-		wrapper.world.New()
```

Comment:
We weren't doing anything useful within `world.New()`, so removed the function entirely.